### DISCUSSION TITLE
I can't flash

### DISCUSSION THREAD

**Discussion by Wuehli** [jomjol/AI-on-the-edge-device#2274] [General]
Hi Guys,

it's so frustating, i can't flash my ESP-CAMs.
I have two, one spare one that worked one (and stopped).
Regardless of which im trying to flash it is never working afterwards.
I tried the espressif windows tool, it's always finishing but not booting afterwards.
If i try the web version i only get this (until i press the boot button).
But afterwards is also not flashing, its preparing very long and then failing.

![grafik](https://user-images.githubusercontent.com/88660266/229199041-94cf9994-ad4c-48b7-87f9-273a4a382538.png)

any ideas?
thanks!
regards.

**Comment by pimp1310** [comment]
hi

what does your wiring look like when flashing?
did you bridge PIN3 / 4   (IOT auf GND) when flashing?
and i flash always with a fdti adapter and set the adapter to 5V

![verkabelung](https://user-images.githubusercontent.com/6306057/229207409-00acd66a-1578-4fff-a41a-3fd11ec3d183.jpg)

and I had to take the GND right next to the 5V, not the one opposite

than you can try it with "flash_download_tool_3.9.4" this works by me like a charm. but test the baudrates. not every works

**Comment by Wuehli** [comment]
![grafik](https://user-images.githubusercontent.com/88660266/229209697-027324ed-23c5-4773-8887-ca4bc3da6510.png)

That is what i always get, but afterwards i see the boot error from the screenshot before.
My wiring is like your picture (checked and changed 1000x times), i also tried additional power supply 5V 3A just to be shure.

**Reply by pimp1310** [reply]
you must select the files in the window on the right screen, and than click "start"

![help](https://user-images.githubusercontent.com/6306057/229217607-130051cf-2a83-4340-892a-27d10d04a91a.jpg)

than the the progress bar comes and he flashes, after a time the FINISH comes again and you can unplug your device.

and i see no error on your screens.

**Comment by Wuehli** [comment]
![4b248e76660ba7e622ccb8e65bfb37e7](https://user-images.githubusercontent.com/88660266/229296953-45fff070-f626-4552-bc40-b8075ad70f51.gif)

Thanks to you i could end the day yesterday in a conciliatory way.

But i still wonder why the webflasher didn't work.

**Reply by caco3** [reply]
Do you have a log or screenshot of the web installer?

**Comment by Wuehli** [comment]
🤦‍♂️

Thanks! This lets me end the day on a conciliatory note.



31.03.2023 21:58:04 Marc ***@***.***>:
